### PR TITLE
Fix #83 - Add inital content for October 2020

### LINKED ISSUE #83: October 2020 Homepage Content Updates

### PR DESCRIPTION
**Pull request by maxxcrawford**
TODO: 
- [x] Optimize Images 

Missing source images: 

- [x] Facebook Container (3:2) 
- [x] Monitor (3:2)

### CODE REVIEW

**Comment by maxxcrawford** [comment]
@stephaniehobson Tagging you for review with the caveat of two images that will need fixing before final green light. Thanks!

File: content/home-2020-oct/card_10/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_10/en-US.md]: We usually add UTMs when linking to other mozilla.org domains. `?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card`

File: content/home-2020-oct/card_11/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_11/en-US.md]: card 11

File: content/home-2020-oct/card_12/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_12/en-US.md]: card 12

(I don't know why we put the card numbers in the YAML, this convention started before me and causes me exactly the same problem(

File: content/home-2020-oct/card_13/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_13/en-US.md]: card 13

File: content/home-2020-oct/card_3/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_3/en-US.md]: UTMs here too please. So I guess mozilla.org and firefox.com domains get them.

File: content/home-2020-oct/card_11/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_11/en-US.md]: No UTMs on links to **www**.mozilla.org. Adding them when linking to the same website causes problems with analytics.

File: content/home-2020-oct/card_8/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_8/en-US.md]: Only one URL here please ;)

File: content/home-2020-oct/card_9/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_9/en-US.md]: UTMs plz

File: content/home-2020-oct/card_5/en-US.md
Comment (stephaniehobson) [inline content/home-2020-oct/card_5/en-US.md]: IMHO this should go to /new but I'll leave it up to you to talk to MJ or not or maybe you already have.

File: content/home-2020-oct/card_12/en-US.md
Comment (maxxcrawford) [reply content/home-2020-oct/card_12/en-US.md]: Heh. Thanks for catching these. 🙈

File: content/home-2020-oct/card_8/en-US.md
Comment (maxxcrawford) [reply content/home-2020-oct/card_8/en-US.md]: MJ clarified. https://foundation.mozilla.org/en/blog/tags/misinformation-monday/

File: content/home-2020-oct/card_5/en-US.md
Comment (maxxcrawford) [reply content/home-2020-oct/card_5/en-US.md]: Updated to https://www.mozilla.org/en-US/firefox/new/!

File: content/home-2020-oct/card_12/en-US.md
Comment (pmac) [reply content/home-2020-oct/card_12/en-US.md]: > (I don't know why we put the card numbers in the YAML, this convention started before me and causes me exactly the same problem(

I'm pretty sure it's my fault. My script that I wrote to extract the data out of the existing page and into the first set of these files added it so I could see it easily in my editor since all the file names were just `en-US.md`. It's definitely not a requirement for them to stay.

Comment (stephaniehobson) [reply content/home-2020-oct/card_12/en-US.md]: Thanks for the history @pmac. I might drop them in my next update.

**Review by stephaniehobson — APPROVED** [review]
Fast turn around for your first time. ⚡  R+ 📰

**Pull request merged**
